responses/nl/HassGetState.yaml

```diff
@@ -1,4 +1,52 @@
 language: nl
 responses:
   intents:
-    HassGetState: {}
+    HassGetState:
+      one: "{{ slots.name | capitalize }} is {{ state.state_with_unit }}"
+      one_yesno: |
+        {% set translations =
+          {
+            "on": "aan",
+            "off": "uit",
+            "open": "open",
+            "closed": "dicht",
+            "opening": "aan het openen",
+            "closing": "aan het sluiten"
+          }
+        %}
+        {% if state.state_with_unit in translations.keys() %}
+          {% set translated = translations[state.state_with_unit] %}
+        {% endif %}
+        {% if query.matched: %}
+          Ja
+        {% else: %}
+          Nee, {{ slots.name }} is {{ translated if translated is defined else state.state_with_unit }}
+        {% endif %}
+      any: |
+        {% if query.matched: %}
+        Ja, {{ query.matched | map(attribute="name") | sort | join(", ") }}
+        {% else: %}
+        Nee
+        {% endif %}
+
+      all: |
+        {% if not query.unmatched: %}
+        Ja
+        {% else: %}
+        Nee, niet {{ query.unmatched | map(attribute="name") | sort | join(", ") }}
+        {% endif %}
+
+      which: |
+        {% if not query.matched: %}
+          Geen
+        {% else: %}
+          {% set match = query.matched | map(attribute="name") | sort | list %}
+          {% if match | length > 3 %}
+            {{ match[:3] | join(", ") ~ " en nog " ~ (match | length - 3)  }}
+          {% else %}
+            {{ match[:-1] | join(", ") ~ " en " ~ match[-1] if match | length > 1 else match[0] }}
```

Comment:
leesbaarder alternatief template format is om de loop.last variable te gebruiken

https://stackoverflow.com/a/11974399

Reply:
Zal ik eens naar kijken, ken het alleen in een for loop :)

Reply:
Oh, ik zie dat het ook in een for loop is.
Maar onderstaande doet hetzelfde als `match | join(", ")` Ik zie niet echt het voordeel van het gebruiken van een for loop.
```jinja
{% set match = ["Slaapkamerlamp",  "Keukenlamp", "Werkbank"] %}
{%- for i in match -%}
  {{ i ~ ", " if not loop.last else i }}
{%- endfor -%}
``` 

Juist omdat ik `Slaapkamerlamp, Keukenlamp en Werkbank` wil (met de `en` tussen de laatste en een-na-laatste) heb ik het zo gedaan. Met de for loop wordt dat ingewikkeld omdat ik dan niet `loop.last` moet hebben, maar ook geen komma wil bij degene daarvoor.

Tenzij ik hier iets slims over het hoofd zie hoor :)

Reply:
Oke, dit werkt wel
```jinja
{%- set match = ["Slaapkamerlamp", "Keukenlamp", "Werkkamerlamp"] -%}
{%- for name in match -%}
  {%- if loop.first -%}
    {{ name }}
  {%- elif loop.last -%}
    {{ " en " ~ name }}
  {%- else -%}
    {{ ", " ~ name }}
  {%- endif -%}
{%- endfor -%}
```

Reply:
Ik dacht meer zoiets:
```jinja2
{%- set match = ["Slaapkamerlamp", "Keukenlamp", "Werkkamerlamp"] -%}
{%- for name in match -%}
  {%- if not loop.first and not loop.last -%}
    ,
  {%- elif loop.last -%}
    en
  {%- endif -%}
  {{ name }}
{%- endfor -%}
```